src/components/BottomSheetInLineNotification.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import { StyleSheet } from 'react-native'
+import Animated, { useAnimatedStyle, useSharedValue } from 'react-native-reanimated'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import InLineNotification, { InLineNotificationProps } from 'src/components/InLineNotification'
+import { useShowOrHideAnimation } from 'src/components/useShowOrHideAnimation'
+import Colors from 'src/styles/colors'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+
+interface Props extends InLineNotificationProps {
+  showNotification: boolean
+}
+
+// this value is used to animate the notification on and off the screen. the
+// value should be equal or greater than the height of the notification,
+// otherwise the notication will not transition smoothly.
+const NOTIFICATION_HEIGHT = 500
```

Comment:
I see, I guess we could retrieve the real height instead of hardcoding, but probably ok for now.

Reply:
i've done [a similar trick](https://github.com/valora-inc/wallet/blob/01a6549621a99097c82997aaf44236d6f624732d/src/home/NotificationCenter.tsx#L220), seems work fine

Reply:
🙈 i am not convinced that we need that level of complexity here (it'd require a state variable and layout measure, sounds like not much but still something) - 500 pixels is over 50% of the screen height of an iphone 13 pro, even more on a smaller device. it's really unlikely that this toast would exceed that height to make the animation janky